quidel_covidtest/tests/test_run.py

```diff
@@ -68,6 +68,15 @@ def test_output_files(self, clean_receiving_dir):
         )
         assert (df.columns.values == ["geo_id", "val", "se", "sample_size"]).all()
 
+        df = pd.read_csv(join("./receiving", "20200718_county_covid_ag_raw_pct_positive.csv"))
+        # TODO: should megacounty 51000 be here (it currently is)?
+        assert set(df.geo_id) == set([51143, 51059, 51000])
+
+        df = pd.read_csv(join("./receiving", "20200718_county_covid_ag_smoothed_pct_positive.csv"))
+        # TODO: should megacounty 51000 be here (it currently isn't)?
+        assert set(df.geo_id) == set([51019, 51143, 51059])
```

Comment:
@jingjtang I wanted to do the same test for smoothing. `df` here looks like 

```
   geo_id        val        se  sample_size
0   51000  49.034241  7.069749         50.0
1   51019  52.897278  7.059187         50.0
2   51059  40.163934  6.328838         60.0
3   51143  50.000000  6.250000         64.0
```

- ✔️ZIP 24534, FIPS 51083 has 4 counts, so it's below threshold and absent.
- ✔️ZIP 24529, FIPS 51117 has 24 counts, so it's below threshold and absent.
- ❓ MEGAFIPS 51000 is present because of aggregating the two above 4 + 24 > 25 = MIN_OBS/2; however, sample_size of 50 seems incorrect - because the sample size was only 29. Probably a bug because it assumes sample_size = MIN_OBS?
- ✔️ZIP 24526, FIPS 51019 has 49 counts, so it's geopooled up to 50.
- ✔️ZIP 24527, FIPS 51143 has 64 counts, so it's above threshold and present.
- ✔️ZIP 22079, FIPS 51059 has 60 counts, so it's above threshold and present.